lib/jekyll-feed.rb

```diff
@@ -12,6 +12,14 @@ def config
       @context.registers[:site].config
     end
 
+    def path
+      if config["feed"] && config["feed"]["path"]
```

Comment:
It's not up to Jekyll-Core it's up to the author of the plugin or in this case @orderedlist, if he wants a hash then that's on him tbh, Jekyll doesn't enforce like that (not that I've seen at least,) so it's entirely up to the author if he wants to blow up because he wants a hash or not, it's not up to Jekyll-Core whether he accepts both gracefully.